src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -82,6 +82,40 @@ std::string postUpdateContext
     return answer;
   }
 
+  //
+  // If upcr.contextElementResponseVector contains >= TWO responses and the second of them is a 'SccFound',
+  // then skip the first, the one that isn't 'SccFound'.
+  // Also, if the first of them is 472, then skip the second
+  //
+  // FIXME P6: Temporary hack for 'Entity found but Attribute not found':
+  //           See FIXME P6 'Temporary hack ...' 
+  //           in MongoCommonUpdate.cpp, function processContextElement
+  //
+
```

Comment:
I agree there are many cases in which the hack doesn't work. However, if we can assure the limitation of it works when the updateContext includes only 1 CE and *doesn't break existing functionality* (which is what my "nested ifs" proposa is about) I think we can be happy.